java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/LoadDataPlan.scala

```diff
@@ -54,42 +81,137 @@ object LoadDataPlan {
     // quote: '\0'(means no quote, the same with spark quote "empty string")
     options += ("header" -> "true")
     options += ("nullValue" -> "null")
+    updateOptionsMap(options, node.GetOption("delimiter"), "sep", getStr)
+    updateOptionsMap(options, node.GetOption("header"), "header", getBool)
+    updateOptionsMap(options, node.GetOption("null_value"), "nullValue", getStr)
+    updateOptionsMap(options, node.GetOption("quote"), "quote", getStr)
 
-    option = node.GetOption("delimiter")
-    if (option != null) {
-      options += ("sep" -> option.GetStr())
-    }
-    option = node.GetOption("header")
-    if (option != null) {
-      // boolean to str
-      options += ("header" -> option.GetBool().toString)
-    }
-    option = node.GetOption("null_value")
-    if (option != null) {
-      options += ("nullValue" -> option.GetStr())
-    }
-    option = node.GetOption("quote")
-    if (option != null) {
-      options += ("quote" -> option.GetStr())
+    // write mode/save mode, not the read mode
+    val modeStr = parseOption(node.GetOption("mode"), "error_if_exists", getStringOrDefault).toLowerCase
+    val mode = modeStr match {
+      case "error_if_exists" => "errorifexists"
+      // append/overwrite, stay the same
+      case "append" | "overwrite" => modeStr
+      case others: Any => throw new UnsupportedHybridSeException(s"unsupported write mode $others")
     }
 
-    (format, mode, options.toMap)
+    // if symbolic link(aka slk)
+    val slk = parseOption(node.GetOption("slk"), "false", getBoolOrDefault).toBoolean
```

Comment:
May use the term `deep_copy` instead of `slk`.

Reply:
ok